src/applications/vaos/tests/mocks/helpers.v2.js

```diff
@@ -484,3 +485,81 @@ export function mockClinicsApi({
 
   return baseUrl;
 }
+
+/**
+ * Mock the browser geolocation api with a given position
+ *
+ * @export
+ * @param {Object} [params]
+ * @param {number} [latitude=53.2734] Latitude value (defaulted to San Diego)
+ * @param {number} [longitude=-7.77832031] Longitude value (defaulted to San Diego)
+ * @param {boolean} [fail=false] Should the geolocation request fail
+ */
+export function mockGetCurrentPosition({
+  latitude = 53.2734, // San Diego, CA
+  longitude = -7.77832031,
+  fail = false,
+} = {}) {
+  global.navigator.geolocation = {
+    getCurrentPosition: sinon.stub().callsFake(
+      (successCallback, failureCallback) =>
+        fail
+          ? Promise.resolve(
+              failureCallback({
+                code: 1,
+                message: 'User denied Geolocation',
+              }),
+            )
+          : Promise.resolve(
+              successCallback({ coords: { latitude, longitude } }),
+            ),
+    ),
+  };
+}
+
+/**
+ * Mocks request to VA community care providers api, used in community care request flow
+ *
+ * @export
+ * @param {Object} address Facility address object with latitude and longitude properties
+ * @param {Array<string>} specialties Array of specialty codes used for a type of care
+ * @param {Array<string>} bbox Array of bounding box coordinates to search in
+ * @param {Array<PPMSProvider>} providers Array of providers to return from mock
+ * @param {boolean} [vaError=false] If true mock will return an error response
+ * @param {number} [radius=60] Miles radius to search within for the mock, used in query param
+ */
+export function mockCCProviderFetch(
```

Comment:
This is sort of fascinating as it's one of the few instance of `v1` API I have seen in our codebase. As far as I can tell, this is still being used in production code but just wanted to confirm whether that's expected?

Reply:
@JunTaoLuo I believe the `v1` here is different than the VAOS API v0/v2 because it's the facilities API (from Lighthouse) which is not part of the VAOS API.

Reply:
I see that makes sense, thanks!